docs/get-started/activities.md

```diff
@@ -0,0 +1,241 @@
+(activities)=
```

Comment:
This isn't a full rewrite. i moved this file from rst to myst md. then i edited it!